Issue of selecting text nodes in mults widgets

juzen2003 opened the pull request:
- Were any Django, import pipeline, table_schema, or dictionary files modified? N
  - Database used: opus3_test_200107
  - All Django tests pass: Y
- Were any JavaScript or CSS files modified? Y
  - JSHINT run on all affected files: Y
  - Tested on Chrome, Firefox, Safari, and iOS: Y
    (Remember to test all browser sizes)
  - Tested for race conditions (with delayed API calls if applicable): NA

Description of changes:
Make sure we can click and drag to copy input and hints text nodes in mults widgets. Default mousedown behavior will not be disabled when clicked targets are input and hints text nodes.

Known problems:
None

juzen2003 committed:
Make sure we don't remove the default mousedown behavior when clicking on
text nodes in mults widgets. This will make sure we can click and drag to
copy choices and hints in mults widgets.
- In search.js (line 114-115), make sure we don't remove the default mouse
  mousedown behavior when clicking on text nodes in mults widgets.
- In search.js (line 1171), assign "op-choice-hints-text" as the class of
  the span tag that wraps the hints text node in mults.

juzen2003 committed:
Clean up code style at line 1166, 1170 in search.js (change '' to "").

rfrenchseti reviewed (changes requested):
Can you also make it so the min/max/null hints are selectable?

juzen2003 committed:
Make all hints description (Min/Max/Nulls) and numbers selectable, and
allow copying of these info.
- In search.js (line 1188), rename "op-choice-hints-text" to "op-hints-info".
- In search.js (line 1132,1139-1144,1156-1161,949-954), wrap Min/Max/Nulls
  with span tags that has class "op-hints-description". This class name
  will be used to enable user selection and caret cursor for these texts.
- In opus.css (line 2293,2674-2677) enabled text selection and caret cursor
  for Min/Max/Nulls in ranges widgets.

juzen2003 committed:
Clean up code style in search.js (line 113-117).

juzen2003 committed:
Make sure hints number is still selectable when it's "?".
- In search.js (line 959, 1232), add op-hints-info to the span tag that
  wraps hints number.
